configure.ac

```diff
@@ -32,6 +32,33 @@ AM_CONDITIONAL([DEVELOPER], [test "x$enable_developer" = "xyes"])
 
 AC_CANONICAL_HOST
 
+AC_ARG_WITH([harfbuzz],
+            AS_HELP_STRING([--without-harfbuzz], [Compile without HarfBuzz support for shaping backend]))
+AM_CONDITIONAL([HARFBUZZ], [test "x$with_harfbuzz" != "xno"])
+
+AM_COND_IF([HARFBUZZ], [
+    PKG_CHECK_MODULES(HARFBUZZ, [harfbuzz >= 2.7.4], [
+        PKG_CHECK_MODULES(FONTCONFIG, fontconfig, [
+            ], [
+                AC_MSG_FAILURE([Harfbuzz found but not fontconfig])
```

Comment:
Those two checks should be decoupled, the code that uses HarfBuzz and the code that uses FontConfig are not dependent and one should be able to build without FontConfig (e.g. on macOS). I’d make at least one font manager a requirement, so at least FontConfig or AppKit.

Reply:
Roger that, I was wondering while I did that refactoring...